test/Microsoft.AspNetCore.Server.Kestrel.Performance/FrameParsingOverheadBenchmark.cs

```diff
@@ -60,14 +60,14 @@ private void ParseRequest()
 
             if (!_frame.TakeStartLine(_buffer, out var consumed, out var examined))
             {
-                RequestParsing.ThrowInvalidRequestLine();
+                RequestParsingBenchmark.ThrowInvalidRequestLine();
```

Comment:
While you're at it, let's move the `Throw`* methods into a separate class (`ErrorUtilities` or something).